rdflib/plugins/parsers/trig.py

```diff
@@ -104,6 +104,8 @@ def graph(self, argstr: str, i: int) -> int:
         self._reason2 = becauseSubGraph
         # type error: Incompatible types in assignment (expression has type "Graph", variable has type "Optional[Formula]")
         self._context = self._store.newGraph(graph)  # type: ignore[assignment]
+        if self._context is not None and self._parentContext is not None:
```

Comment:
`self._store.newGraph(graph)` always returns a value so `self._context is not None` should always be true and can be omitted as far as I can tell.

This can also move up a bit, I think it would be best at line 100 here, and then it can also only use `self._context` it won't be moved to `self._parentContext` yet.

https://github.com/RDFLib/rdflib/blob/6f53455c788b123826ad7321c7cea5f381ae7a77/rdflib/plugins/parsers/trig.py#L99-L101

Reply:
Fixed now.